custom_components/ramses_cc/binary_sensor.py

```diff
@@ -152,22 +147,22 @@ class RamsesFaultLog(RamsesBinarySensor):
     @property
     def available(self) -> bool:
         """Return True if the device has been seen recently."""
-        if msg := self._device._msgs.get("0418"):
+        if msg := self.rf_entity._msgs.get("0418"):
             return dt.now() - msg.dtm < timedelta(seconds=1200)
 
     @property
     def extra_state_attributes(self) -> dict[str, Any]:
         """Return the integration-specific state attributes."""
-        return {
-            "active_fault": self._device.tcs.active_fault,
-            "latest_event": self._device.tcs.latest_event,
-            "latest_fault": self._device.tcs.latest_fault,
+        return super().extra_state_attributes() | {
+            "active_fault": self.rf_entity.tcs.active_fault,
+            "latest_event": self.rf_entity.tcs.latest_event,
+            "latest_fault": self.rf_entity.tcs.latest_fault,
```

Comment:
I am afraid I don't see the value in renaming this attr to `rf_entity`. 

In addition, it should also start with an underscore as it is not for HA, only for self.

I suggest leaving it as `_device`.